addon/components/docs-viewer/x-nav/component.js

```diff
@@ -17,19 +18,7 @@ export default Component.extend({
   store: service(),
   packageJson,
 
-  addonLogo: computed(function() {
-    let name = packageJson.name;
-    let logo;
-    if (name.match(/ember-cli/)) {
-      logo = 'ember-cli';
-    } else if (name.match(/ember-data/)) {
-      logo = 'ember-data';
-    } else {
-      logo = 'ember';
-    }
-
-    return logo;
-  }),
+  addonLogo: addonLogo(packageJson),
```

Comment:
Unsure if you prefer passing `packageJson` into the CP macro or would rather `this.get('packageJson')` within the CP definition. I like this API better since it makes less assumptions about that `packgeJson` property being available.